### PR TITLE
[1LP][RFR][NOTEST]Vm name enhancement

### PR DESCRIPTION
**Pull request by kedark3**
## Purpose or Intent
- __Enhancement__ to replace one off fixtures and create_on_provider calls with `create_vm` fixture.

NO PRT RUN. Will run in Jenkins.

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 6bdb1ba44ceece66db2ece937ff1d7777c3f1d8e

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`ansible_policy_linked_vm`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 1]
```diff
@@ -99,33 +98,19 @@ def ansible_linked_vm_action(appliance, local_ansible_catalog_item, new_vm):
...
-
-@pytest.fixture()
-def ansible_policy_linked_vm(appliance, new_vm, ansible_linked_vm_action):
+def ansible_policy_linked_vm(appliance, create_vm, ansible_linked_vm_action):
     policy = appliance.collections.policies.create(
         VMControlPolicy,
         fauxfactory.gen_alpha(15, start="policy_"),
-        scope="fill_field(VM and Instance : Name, INCLUDES, {})".format(new_vm.name),
+        scope="fill_field(VM and Instance : Name, INCLUDES, {})".format(create_vm.name),
     )
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: **optional:** I would have used fstring here

**Comment by john-dupuy** [comment]
I think before merge we should do a Jenkins run `without` `--composite-uncollect` for this and #9851

**Comment by jawatts** [comment]
@john-dupuy agree

**Pull request merged**